src/System.Net.Requests/pkg/System.Net.Requests.pkgproj

```diff
@@ -2,14 +2,8 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\3.9.0\System.Net.Requests.depproj">
-      <SupportedFramework>wp8</SupportedFramework>
-    </ProjectReference>
-    <ProjectReference Include="..\ref\4.0.0\System.Net.Requests.depproj">
-      <SupportedFramework>net45;netcore45;wpa81</SupportedFramework>
-    </ProjectReference>
     <ProjectReference Include="..\ref\System.Net.Requests.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Historically, it was advantageous to keep contracts in as low a 'netstandard' as possible to provide for the widest possible set of platforms supported. That is why, for example, System.Net.Http, was OOB'd in part because adding APIs would normally kick it out of 'netstandard13' into something newer. Instead, we were able to OOB the assembly for Desktop to avoid bumping it out of 'netstandard13'.

So, that thinking in general applies to how we traditionally look at contracts and revising contracts. Will it need to move to a new 'netstandard'.  That was the guidance originally provided by @ericstj and @weshaggard .

Perhaps with the big push toward .NET Standard 2.0, it implies that folks have to accept these new rules for netstandard17. But in general, we like to know what new dependencies we are taking on and the reason we have to bump to a newer netstandard.  Pure OOB packages/libraries that were never in .NET Framework (Desktop) can continue (usually) to rev the contract version (and get more APIs) without necessarily bumping 'netstandard' version.